components/Avatar/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="avatar.css">
+    <link rel="stylesheet" href="/components/sideNav/sideNav.css">
+    <link rel="icon" type="image/png" href="https://monkey-group.be/wp-content/uploads/2020/11/Favicon-2.png"/>
+    <title>Avatar</title>
+</head>
+<body>
+    <nav class='navigation-bar' >
+        <div class='list-parent-left' id='symbol'>
+            <a class='li-link'href='../../index.html'> <h3>Chikku<span class='heading-color'>Ui</span></h3> </a>  
+        </div>
+        <div class='list-parent-right' id='modal'>
+            <a  href="javascript:void(0);" class="li-icon" onclick="modelFunction()">
+                <i class="fa fa-bars"></i>
+              </a>
+              <a class='li-link' href='../../index.html'> Home</a>
+              <a class='li-link' href='index.html'>Documentation</a>
+            
+              <a class='li-link' target='_blank' href='https://github.com/vineeth-t/Main/tree/Devlopment'>Github</a>
+          
+</div>
+        </nav>
+        <header class='header' >
+            <div class='menu-list' id='menu-items'>
+                <a class="li-link component" href='/components/Alert/index.html'>Alert</a>
```

Comment:
The class `component` here, is not explaining the purpose of it. You can try something, that describes the functioning of the class better.